plugin/log/README.md

```diff
@@ -23,20 +23,21 @@ log
 Or if you want/need slightly more control:
 
 ~~~ txt
-log [NAME] [FORMAT]
```

Comment:
If two parameters where specified we already assume the last one if the format - that should also make this change smaller (I guess).

Looking at the code and docs I see we also don't mention the use of `{common}` or `{combined}` as a shortcut. I'll send a PR for that.

Reply:
No, we just say the last one is `FORMAT`, this is happening now anyway in the code so there would be no change